src/main/webapp/app/assessment-instructions/assessment-instructions.component.html

```diff
@@ -10,11 +10,8 @@ <h3 class="card-title">
         </div>
         <div class="card-body markdown-preview">
             <h3>{{ exercise.title }}</h3>
-            <br />
-            <jhi-expandable-paragraph header="{{ 'assessmentInstructions.problemStatement' | translate }}" [text]="formattedProblemStatement"></jhi-expandable-paragraph>
-            <br />
+            <jhi-expandable-problem-statement [exercise]="exercise"></jhi-expandable-problem-statement>
             <jhi-expandable-sample-solution [exercise]="exercise"></jhi-expandable-sample-solution>
```

Comment:
The `<jhi-expandable-sample-solution />` Component only takes a `ModelingExercise` object as an `@Input`. Please check out changes in #1108 related to assessment instructions. There will be a merge conflict.